src/pages/LabEditor/LabEditor.tsx

```diff
@@ -105,6 +108,24 @@ export default function LabEditor({match: {params: {moduleId, labId}}}: Props) {
   const getFieldName = (name: keyof LabForm) => name;
 
   const [selectedVm, setSelectedVm] = useState<number|undefined>();
+  const popover = (
+    <Popover id='popover-basic' style={{maxWidth: '750px'}}>
+      <Popover.Title as='h3'>Info:</Popover.Title>
+      <Popover.Content>
+        <div>- Temporary - only persist for only an hour and torn down afterwards. Their is no long term persistence with temporary labs.</div>
+        <div>
+          - Semester - these labs persist until a certain date-time is reached. 
+          Once the date-time is reach the lab is torn down is unable to be started again.
+        </div>
+        <div>- Permanent - these labs are never torn down.</div>
+      </Popover.Content>
+    </Popover>
+  );
+  const Info = () => (
+    <OverlayTrigger trigger='hover' placement='right' overlay={popover}>
+      <a><FontAwesomeIcon icon={faInfoCircle}/></a>
+    </OverlayTrigger>
+  );
```

Comment:
Can you move this code into a separate component and call it here? This will help keep this code file smaller.

Reply:
Done. Moved popover code to a separate component and called it from `LabEditor.tsx`.